README.md

```diff
@@ -44,23 +44,24 @@ To run the application:
 yarn start
 ```
 
-**Note that some [configuration](#configuration) may be necessary to run the service in isolation**
+**Note that some [configuration](#environment-variables) may be necessary to run the service in isolation**
 
-## Testing
-
-To run all tests:
-```
-yarn test
-```
-
-## Configuration
+## Environment Variables
 
 The following environment variables can be configured.
 
 | Name | Description | Required |
 | --- | --- | --- |
 | `EQ_SCHEMA_VALIDATOR_URL` | The URL of the schema validation service. See [Running with Docker Compose](#running-with-docker-compose). | Yes |
 | `EQ_AUTHOR_API_URL` | The URL of the GraphQL API server. If not supplied, mock API will be used. | Yes |
```

Comment:
Think this line about the mock api can be taken out, pretty sure if the API_URL isn't provided it'll throw.

Reply:
Yep, looks like you're right

if (isNil(process.env.EQ_AUTHOR_API_URL)) {
    throw Error("EQ_AUTHOR_API_URL not specified");
}

Ill remove that